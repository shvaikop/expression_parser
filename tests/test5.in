9
xyz= 51
$XyZ
$xYZ
AB = $XYZ + 7 % 7
XYz= 67
xYz= $ab
  a= 78
a =67
a= xyz
 
