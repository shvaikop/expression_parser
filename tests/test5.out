51
51
51
invalid
invalid
invalid
invalid
invalid
invalid
